ts/src/bitforex.ts

```diff
@@ -381,7 +381,7 @@ export default class bitforex extends Exchange {
             request['limit'] = limit;
         }
         if (since !== undefined) {
-            request['startTime'] = since;
+            request['startTime'] = Math.max (since - 1, 0);
```

Comment:
@ndubel nice catch 🙂, probably it does a `>` instead of a `>=` comparison

Reply:
yes, some exchanges have such behavior. In ccxt we use >=